### PR TITLE
Fixes based in feedbacks from story review

### PR DESCRIPTION
**Pull request by waldemarnt**
## Before
screenshot from 2016-05-04 21-05-38

## After
screenshot from 2016-05-04 21-09-58
screenshot from 2016-05-04 21-10-32

### CODE REVIEW

File: source/includes/equipment_api/_equipment.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Header key    | Value
 ------------- | -----
 Authorization | Bearer [token]
 
-Refer to the [Authentication](#authentication) section to learn how to aquire a valid token.
+Refer to the [Authentication](#how-to-authenticate) section to learn how to aquire a valid token.
```
Comment (bltavares) [inline source/includes/equipment_api/_equipment.md]: Should we change the content to `[How to Authenticate]` as well?

File: source/includes/external_user_workflow/_workflow.md [hunk 2]
```diff
@@ -92,31 +92,31 @@ curl -k -X "POST"\
...
-## Making Requests
+### Making Requests
 
-> Get Equipment list from Equipment API
+> Get Equipment list from Equipment API with basic authentication
 
 ```shell
 curl -X GET \
-    -H "Authorization: Bearer 2483055a-cb08-4027-a8a9-03f89988d025"\
+    -u email@domain.com:password \
```
Comment (bltavares) [inline source/includes/external_user_workflow/_workflow.md]: Usually it is better to use `example.com` as this is an official domain name for examples and can't be registered.
`domain.com` is an actual domain owned by a registrar.

File: source/includes/equipment_api/_equipment.md (hunk 1 above)
Comment (jeffhsta) [inline source/includes/equipment_api/_equipment.md]: :+1:

**Comment by bltavares** [comment]
LGTM.
Would you mind adding a screenshot with the changes?

**Comment by jeffhsta** [comment]
@bltavares sure, I can do it

**Pull request merged**